Cesium.CodeGen/Contexts/Meta/FunctionInfo.cs

```diff
@@ -19,6 +20,8 @@ internal record FunctionInfo(
     public StorageClass StorageClass { get; set; } = StorageClass;
     public bool IsDefined { get; set; } = IsDefined;
     public string? CliImportMember { get; set; }
+    public string? DllLibraryName { get; set; }
+    public string? DllImportNameStrip { get; set; }
```

Comment:
Let's add documentation referencing `PInvokeDefinition.Prefix`, for the reader to get why are there two properties.